src/Select/__stories__/Select.stories.js

```diff
@@ -76,6 +77,8 @@ export const dev = () => (
     <Select
         compact={boolean('compact', false)}
         disabled={boolean('disabled', false)}
+        onBlur={() => console.log('on-blur-callback')}
+        onFocus={() => console.log('on-focus-callback')}
```

Comment:
I originally tried to use addon-actions, but that slowed storybook down so much. I think there's a bug there